work_with_builds/using-builds.adoc

```diff
@@ -7,7 +7,7 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 
-After installing {builds-shortname}, you can create a `buildah` or `source-to-image` build for use. You can also delete custom resources that are not required for a build.
+After installing {builds-shortname}, you can create a `buildah`, `source-to-image`, `buildpacks` builds for use. You can also use an Open Container Initiative (OCI) to create your build. You can also delete custom resources that are not required for a build.
```

Comment:
the use of two commas here doesn't look right 

"After installing Builds, you can create a buildah, source-to-image, buildpacks builds for use."

and the singular/plural is odd

you can create a ....  builds for use